test/NPMCaller.t.sol

```diff
@@ -133,7 +153,7 @@ contract NPMCallerTest is BaseTest {
 
     /// @dev Returns the digest used in the permit signature verification
     function permitDigest(address spender, uint256 tokenId, uint256 deadline) internal view returns (bytes32) {
-        (uint96 nonce, , , , , , , , , , , ) = npm.positions(tokenId);
+        (uint96 nonce, , , , , , , , , , , ) = IUniV3NPM(address(npm)).positions(tokenId);
```

Comment:
how does permitDigest work for PCS if using an IUniV3NPM here?

Reply:
The PCS NPM interface is almost identical to UniV3NPM; PCSV3NPM just has an extra `deployer()` function and you can look at 'IPCSV3NonfungiblePositionManager' to see it inherits from the UniV3 NPM. Thus, there is no need to distinguish between UniV3 and PCSV3 here.